go.mod

```diff
@@ -10,12 +10,12 @@ require (
 	go.opentelemetry.io/otel/exporters/metric/prometheus v0.20.0
 	go.opentelemetry.io/otel/metric v0.20.0
 	golang.org/x/crypto v0.0.0-20220722155217-630584e8d5aa
-	golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd
-	google.golang.org/grpc v1.48.0
+	golang.org/x/net v0.0.0-20220722155237-a158d28d115b
```

Comment:
is this a CVE fix? can we add a comment for this?

Reply:
This one is not a CVE fix. I'll add the CVE fix in another PR and also enable trivy to check `library` vulnerabilities.